dao/src/main/kotlin/tables/SnippetsTable.kt

```diff
@@ -66,8 +66,8 @@ class SnippetDao(id: EntityID<Long>) : LongEntity(id) {
                 it.mapToModel().provenance == snippet.provenance && it.additionalData?.data == snippet.additionalData
```

Comment:
Commit message: I did not fully understand the implications on first reading. Maybe, it could be made clearer, e.g. "Therefore, this unique constraint does not make sense, since there will never be duplicates anyway." or something like that?

Reply:
Rephrased.